recipes/thriftpy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: thriftpy
+  version: "0.3.5"
+
+source:
+  git_url: https://github.com/eleme/thriftpy
+  git_rev: v0.3.5
+
```

Comment:
We should add this to the linter. Issue raised in https://github.com/conda-forge/conda-smithy/issues/126